internal/image/image.go

```diff
@@ -11,9 +11,9 @@ import (
 	"path/filepath"
 	"strings"
 
-	"github.com/dghubble/trie"
 	v1 "github.com/google/go-containerregistry/pkg/v1"
 	"github.com/google/go-containerregistry/pkg/v1/tarball"
+	"github.com/google/osv-scanner/internal/image/internal/pathtree"
```

Comment:
is there a way to only have one internal in the path?
like `google/osv-scanner/internal/pathtree` or `google/osv-scanner/internal/image/pathtree`?

Reply:
+1 is there a reason to have a nested `internal` here?

Reply:
I'll remove the second layer of internal, it is originally here since pathtree features are tied quite closely to image scanning.